resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -29,25 +29,27 @@ import org.scalatest.time.{Minutes, Seconds, Span}
 import scala.collection.JavaConverters._
 
 import org.apache.spark.SparkFunSuite
+import org.apache.spark.deploy.k8s.integrationtest.TestConfig._
 import org.apache.spark.deploy.k8s.integrationtest.backend.{IntegrationTestBackend, IntegrationTestBackendFactory}
-import org.apache.spark.deploy.k8s.integrationtest.config._
-import org.apache.spark.launcher.SparkLauncher
 
 private[spark] class KubernetesSuite extends SparkFunSuite
-  with BeforeAndAfterAll with BeforeAndAfter {
+  with BeforeAndAfterAll with BeforeAndAfter with BasicTestsSuite with SecretsTestsSuite
+  with PythonTestsSuite {
```

Comment:
In the future if we dont like this pattern we can just create autonomous test suits and scala test will pick them up. This is a starting point for separating tests in different files.

Reply:
This seems a bit cumbersome but is advantageous should we add a flag in our PRB for triggering just base tests, + python, +r, and for the Kerberos integration tests (which are quite sizeable) + Kerberos. I like the decision to separate tests. I think it is appropriate.

Reply:
Where you were thinking of putting the flag?

Reply:
Probably a mvn flag i.e. —python, —r, —kerberos

Reply:
I think we can do that in another PR right?

Reply:
SGTM